airflow/datadog_checks/airflow/airflow.py

```diff
@@ -0,0 +1,55 @@
+# (C) Datadog, Inc. 2019
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import requests
+
+from datadog_checks.base import AgentCheck, ConfigurationError
+
+AIRFLOW_STATUS_OK = "OK"
+
+
+class AirflowCheck(AgentCheck):
+    def check(self, instance):
+        tags = instance.get('tags', [])
+
+        base_url = instance.get('url')
+        if not base_url:
+            raise ConfigurationError('Missing configuration: url')
```

Comment:
Let's do https://github.com/DataDog/integrations-core/blob/e40678822a5ad48496089f8733da865158adb72a/vault/datadog_checks/vault/vault.py#L62 then https://github.com/DataDog/integrations-core/blob/e40678822a5ad48496089f8733da865158adb72a/vault/datadog_checks/vault/vault.py#L81-L84

Reply:
👍 Sure, refactored. Let me know if it's ok.

Good to only parse config once if it succeed.